packages/suite/src/actions/suite/guideActions.ts

```diff
@@ -0,0 +1,62 @@
+import qs from 'qs';
+import { GUIDE } from './constants';
+import type { ActiveView, FeedbackPayload, FeedbackType } from '@suite-types/guide';
+import { Dispatch } from '@suite-types';
+import { addToast } from '@suite-actions/notificationActions';
+import { getLocationHostname } from '@suite-utils/env';
+
+export type GuideAction =
+    | { type: typeof GUIDE.OPEN }
+    | { type: typeof GUIDE.CLOSE }
+    | { type: typeof GUIDE.SET_VIEW; payload: ActiveView }
+    | { type: typeof GUIDE.OPEN_ARTICLE; payload: string };
+
+export const open = () => ({
+    type: GUIDE.OPEN,
+});
+
+export const close = () => ({
+    type: GUIDE.CLOSE,
+});
+
+export const setView = (payload: ActiveView) => ({
+    type: GUIDE.SET_VIEW,
+    payload,
+});
+
+export const openArticle = (payload: string) => (dispatch: Dispatch) => {
+    dispatch(setView('GUIDE_ARTICLE'));
+    dispatch({
+        type: GUIDE.OPEN_ARTICLE,
+        payload,
+    });
+};
+
+const getUrl = (feedbackType: FeedbackType) => {
+    const typeUri = feedbackType === 'BUG' ? 'bugs' : 'feedback';
+    const base = `https://data.trezor.io/suite/${typeUri}`;
+    const hostname = getLocationHostname();
+    switch (hostname) {
+        case 'beta-wallet.trezor.io':
+        case 'suite.trezor.io':
+            return `${base}/prod.log`;
+        case 'localhost':
+        case 'staging-suite.trezor.io':
+        default:
+            return `${base}/develop.log`;
+    }
+};
+
+export const sendFeedback = ({ type, payload }: FeedbackPayload) => (dispatch: Dispatch) => {
+    const url = getUrl(type);
+    const params = qs.stringify(payload);
+    try {
+        fetch(`${url}?${params}`, {
```

Comment:
not sure, but isn't fetch async?

```suggestion
        await fetch(`${url}?${params}`, {
```